summoner-cli/src/Summoner/Project.hs

```diff
@@ -85,16 +85,17 @@ generateProject isOffline projectName Config{..} = do
         (settingsGitHub && not settingsNoUpload)
         (YesNoPrompt "private repository" "Create as a private repository (Requires a GitHub private repo plan)?")
         cPrivate
-    settingsTravis   <- decisionIf settingsGitHub (mkDefaultYesNoPrompt "Travis CI integration") cTravis
-    settingsAppVeyor <- decisionIf settingsGitHub (mkDefaultYesNoPrompt "AppVeyor CI integration") cAppVey
-    settingsIsLib    <- decisionToBool cLib (mkDefaultYesNoPrompt "library target")
-    settingsIsExe    <- let target = "executable target" in
+    settingsGhActions <- decisionIf settingsGitHub (mkDefaultYesNoPrompt "GitHub Actions CI integration") cGhActions
+    settingsTravis    <- decisionIf (settingsGitHub && not settingsGhActions) (mkDefaultYesNoPrompt "Travis CI integration") cTravis
```

Comment:
I chose to prompt for Travis only if Actions CI was declined. I justify this on the basis that (in my experience) Actions CI is significantly faster than Travis, and does not require any setup other than the presence of a workflow .yml file, but I’m happy to change this, or the order of these prompts, if you’d prefer to give Travis primacy.

Reply:
I have no problem changing it to prompting for both Actions and Travis.

Reply:
That would be great, thank you!

Reply:
Done.